relay-config/src/config.rs

```diff
@@ -1341,10 +1341,11 @@ impl Default for CardinalityLimiter {
 #[derive(Serialize, Deserialize, Debug)]
 #[serde(default)]
 pub struct Health {
-    /// Interval in which Relay will refresh system information, like current memory usage.
+    /// Interval in which Relay will refresh internal health checks.
     ///
     /// Defaults to 3 seconds.
-    pub sys_info_refresh_interval_secs: u64,
+    #[serde(alias = "sys_info_refresh_interval_secs")]
```

Comment:
I think it's fine to drop this, stuff like this stays around for too long.

Reply:
👍  Checked now that it's not used, removing